src/workerd/api/http.h

```diff
@@ -944,6 +944,7 @@ class Request: public Body {
         readonly cf?: Cf;
       });
     }
+    JSG_READONLY_PROTOTYPE_PROPERTY(cache, getCache);
```

Comment:
nit: let's keep this grouped with the others above

Reply:
Should I also be splitting this into an instance vs prototype? I'm not exactly sure what line is an appropriate place to put that without splitting it.

Reply:
instance is only here for legacy reasons, anyone who has their compat date set that far back or has that flag turned off likely isn't going to update to support this so I wouldn't bother. Let's keep this as a prototype property.

Reply:
Let me know if this is fixed now